ml-agents/mlagents/trainers/learn.py

```diff
@@ -201,14 +197,16 @@ def run_training(
         summaries_dir = "/{docker_target_name}/summaries".format(
             docker_target_name=options.docker_target_name
         )
-
     trainer_config = load_config(trainer_config_path)
+    port = options.base_port + (sub_id * options.num_envs)
```

Comment:
I think this change is OK, but can you update the `--base-port` docs in https://github.com/Unity-Technologies/ml-agents/blob/a7c1fccd9a86045dece3206604fa93064aac58ea/docs/Training-ML-Agents.md#command-line-training-options to make it clear that it's not used when connecting to the editor? 

(Otherwise I feel it's a minor violation of https://en.wikipedia.org/wiki/Principle_of_least_astonishment, because when I set --base-port=12345, it might have no effect)